src/utils/QCCalculator.cpp

```diff
@@ -156,9 +156,11 @@ class TOPPQCCalculator :
 
   float calculateSNmedian (MSSpectrum<Peak1D>& spec, bool norm = true)
   {
+	if (spec.size() == 0) return 0;
```

Comment:
Looks good. Leads to the question though, why there would be any empty spectrum.